cli/src/lib/validators/validatePackageExports.js

```diff
@@ -6,6 +6,7 @@ const { writeJSON } = require('fs-extra')
  * Ensure that package.main, package.module, and package.exports are valid
  */
 
+const normalizeExtension = ext => ext.replace(/\.[tj]sx?$/, '.js')
```

Comment:
Not necessary, but could be used in the above file as well

Reply:
Yeah, wasn't sure it was worth it but is nicer with shared code - moved to helper module in b0dec7a